From 34b7a516892fda12c61196a1bb7af01e8126d091 Mon Sep 17 00:00:00 2001
From: Stephen <stephen@vamrs.com>
Date: Fri, 6 Sep 2019 15:41:02 +0800
Subject: [PATCH 69/97] overlays-rockpi4: set reg property format

When building, it complains that reg property has invalid length (4 bytes)
(#address-cells == 2, #size-cells == 1).

Signed-off-by: Stephen <stephen@vamrs.com>
---
 .../dts/rockchip/overlays-rockpi4/at24c02-overlay.dts    | 2 ++
 .../overlays-rockpi4/raspberrypi-7-inch-lcd-overlay.dts  | 9 +++++----
 .../dts/rockchip/overlays-rockpi4/spi1-flash-overlay.dts | 2 ++
 3 files changed, 9 insertions(+), 4 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/overlays-rockpi4/at24c02-overlay.dts b/arch/arm64/boot/dts/rockchip/overlays-rockpi4/at24c02-overlay.dts
index 535753163ff0..852cfda11605 100644
--- a/arch/arm64/boot/dts/rockchip/overlays-rockpi4/at24c02-overlay.dts
+++ b/arch/arm64/boot/dts/rockchip/overlays-rockpi4/at24c02-overlay.dts
@@ -9,6 +9,8 @@
 		target = <&i2c2>;
 
 		__overlay__ {
+			#address-cells = <1>;
+			#size-cells = <0>;
 
 			at24c02@50 {
 				status = "okay";
diff --git a/arch/arm64/boot/dts/rockchip/overlays-rockpi4/raspberrypi-7-inch-lcd-overlay.dts b/arch/arm64/boot/dts/rockchip/overlays-rockpi4/raspberrypi-7-inch-lcd-overlay.dts
index a0f475bd8e8b..f64c75782cc8 100644
--- a/arch/arm64/boot/dts/rockchip/overlays-rockpi4/raspberrypi-7-inch-lcd-overlay.dts
+++ b/arch/arm64/boot/dts/rockchip/overlays-rockpi4/raspberrypi-7-inch-lcd-overlay.dts
@@ -19,12 +19,13 @@
 		__overlay__ {
 			rockchip,dual-channel = <&dsi>;
 			status = "okay";
+			#address-cells = <1>;
+			#size-cells = <0>;
 			
 			panel: panel@0 {
-			compatible ="rockpi,tc358762";
-			reg = <0>;
-			status = "okay";
-	
+				compatible ="rockpi,tc358762";
+				reg = <0>;
+				status = "okay";
 			};
 		};
 	};
diff --git a/arch/arm64/boot/dts/rockchip/overlays-rockpi4/spi1-flash-overlay.dts b/arch/arm64/boot/dts/rockchip/overlays-rockpi4/spi1-flash-overlay.dts
index bca5891a8594..5166de70eaec 100644
--- a/arch/arm64/boot/dts/rockchip/overlays-rockpi4/spi1-flash-overlay.dts
+++ b/arch/arm64/boot/dts/rockchip/overlays-rockpi4/spi1-flash-overlay.dts
@@ -13,6 +13,8 @@
 		__overlay__ {
 			status = "okay";
 			max-freq = <10000000>;
+			#address-cells = <1>;
+			#size-cells = <0>;
 
 			spiflash: spi-flash@0 {
 				#address-cells = <0x1>;
-- 
2.25.1

